packages/react-devtools-shared/src/backend/DevToolsComponentStackFrame.js

```diff
@@ -29,12 +29,19 @@ export function describeBuiltInComponentFrame(name: string): string {
       prefix = (match && match[1]) || '';
     }
   }
+  let suffix = '';
+  if (__IS_CHROME__ || __IS_EDGE__) {
+    suffix = ' (<anonymous>)';
+  } else if (__IS_FIREFOX__) {
+    suffix = '@unknown:0:0';
+  }
```

Comment:
Chrome parses the name as the location if it doesn't have one. `<anonymous>` is the only special case that works I think.

Firefox skips lines that don't have location so we need to add a fake one which can be anything really. I don't think there's any special case that doesn't linkify it.

Safari is fine without it.